docs/StardustDocs/d.tree

```diff
@@ -25,7 +30,6 @@
             <toc-element topic="schemasImportOpenApiJupyter.md"/>
         </toc-element>
     </toc-element>
-    <toc-element topic="installation.md"/>
```

Comment:
I think it's likely that we refer to this somewhere (notebook examples, videos, blogposts), did you check with global search in ide? Anyway, we should configure and test redirect https://github.com/Kotlin/dataframe/blob/master/docs/StardustDocs/redirection-rules.xml

Reply:
First question: what is the global search in IDE?
The second question: where to read about redirection?

IMHO, we don't need to be so patient to that before we release stable and full covered documentation

Reply:
https://plugins.jetbrains.com/plugin/20158-writerside/docs/place-redirects.html
I don't mind other topics such as operations etc, but installation page specifically should be redirected. It won't take too much effort